docs/en-US/component/avatar.md

```diff
@@ -47,28 +47,28 @@ avatar/fit
 
 :::
 
-## Avatar API
+## API
 
-### Avatar Attributes
+### Attributes
 
-| Name      | Description                                               | Type                                                       | Default     | Required |
-| --------- | --------------------------------------------------------- | ---------------------------------------------------------- | ----------- | -------- |
-| `icon`    | representation type to icon, more info on icon component. | `string \| Component`                                      | —           | No       |
-| `size`    | avatar size.                                              | `number \| 'large' \| 'default' \| 'small'`                | `'default'` | No       |
-| `shape`   | avatar shape.                                             | `'circle' \| 'square'`                                     | `'circle'`  | No       |
-| `src`     | the source of the image for an image avatar.              | `string`                                                   | —           | No       |
-| `src-set` | native attribute `srcset` of image avatar.                | `string`                                                   | —           | No       |
-| `alt`     | native attribute `alt` of image avatar.                   | `string`                                                   | —           | No       |
-| `fit`     | set how the image fit its container for an image avatar.  | `'fill' \| 'contain' \| 'cover' \| 'none' \| 'scale-down'` | `'cover'`   | No       |
+| Name    | Description                                               | Type                                                              | Default     |
+| ------- | --------------------------------------------------------- | ----------------------------------------------------------------- | ----------- |
+| icon    | representation type to icon, more info on icon component. | ^[string] / ^[Component]                                          | —           |
+| size    | avatar size.                                              | ^[number] / ^[Enum]`'large' \| 'default' \| 'small'`              | `'default'` |
+| shape   | avatar shape.                                             | ^[Enum]`'circle' \| 'square'`                                     | `'circle'`  |
```

Comment:
Enum => enum
Function => function
Everyday types should be kept in lowercase, WDYT?